git.md

```diff
@@ -83,3 +83,51 @@
 - **Apply Stashed Changes**: `git stash apply`
 - **Show Configuration**: `git config --list`
 
+
```

Comment:
- To make it part of notes, probably we should add a new heading called `Knowledge Base` 
- Make the question a heading 4.